### PR TITLE
fix(footer): Improve styleguide trigger positioning

### PR DESCRIPTION
**Pull request by jackmu95**
This commit fixes a positioning issue of the styleguide trigger (heart) icon that adds some additional whitespace beneath the footer as you can see here:

old-styleguide-trigger-desktop

**Improved positioning:**

new-styleguide-trigger-desktop
new-styleguide-trigger-mobile

### CODE REVIEW

**Comment by Foxandxss** [comment]
ping @ericjim

**Comment by ericjim** [comment]
I'll defer to my colleague @amitafr?

**Comment by jackmu95** [comment]
@amitafr: I've added some some more right spacing on desktop in [this](18dba19b8efaa7009007bb5969d42fb2b14c1940) commit and made some more improvements to the icon positioning.

**Comment by jackmu95** [comment]
@all Any progress on reviewing on this?

**Comment by alexwolfe** [comment]
These changes look awesome. Please merged forward so there are not conflicts with master and I'll get this in. 👍

File: public/resources/css/module/_footer.scss [hunk 1]
```diff
@@ -77,33 +77,28 @@
   }
 
   footer {
-    text-align: left;
-    padding: ($unit * 2) ($unit * 6);
+    text-align: center;
```
Comment (alexwolfe) [inline public/resources/css/module/_footer.scss]: Can you keep this left on desktop, center for mobile. Thanks.

**Comment by alexwolfe** [comment]
@jackmu95 Just had one small change that's needed and it's ready. Thanks again for the help.

File: public/resources/css/module/_footer.scss (hunk 1 above)
Comment (jackmu95) [inline public/resources/css/module/_footer.scss]: Will fix that ASAP.

**Comment by jackmu95** [comment]
@alexwolfe What do you think about a color transition of the hear icon to the Angular red?

**Comment by alexwolfe** [comment]
@jackmu95 Yah, I like that idea on hover. If you wanna add that it would be great. Can you use the new color vars. I think you would need something like $red-500 or $red-600.

**Comment by jackmu95** [comment]
@alexwolfe I'am really happy with the hover effect now :)

**Comment by alexwolfe** [comment]
👍  Thanks, nice work. Merging this one 🚢

**Pull request merged**